README.md

```diff
@@ -1,2 +1,21 @@
 # vacanthomes_website
 Vacant Homes - Frontend for the the web
+
+
+## Getting Started
+The website requires a http server for the file reading to work.
+Please note that if you try to open directly from the file, Chrome return an error that only http, https protocols are allowed to read from the file. On my local machine I installed that npm module to run code on the local server 
```

Comment:
Change "return" to "returns" or "will return"

Change "http" to "HTTP" and "https" to "HTTPS"